pki/deploy.go

```diff
@@ -331,7 +331,7 @@ func FetchFileFromHost(ctx context.Context, filePath, image string, host *hosts.
 		Binds:      Binds,
 		Privileged: true,
 	}
-	isRunning, err := docker.IsContainerRunning(ctx, host.DClient, host.Address, containerName, true)
+	isRunning, err := docker.DoesContainerExist(ctx, host.DClient, host.Address, containerName, true)
```

Comment:
It is better to make the variable name matches the actual meaning to reduce confusion.
```suggestion
	exist, err := docker.DoesContainerExist(ctx, host.DClient, host.Address, containerName, true)
```